radis/levels/partfunc.py

```diff
@@ -2286,9 +2344,81 @@ def gi(self, ElecState):
         return gi(M, I)
 
 
+
+    print("Testing parfunc: {0}".format(_run_testcases()))
```

Comment:
`_run_testcases` is not defined. Did you try to run your example for an electronic spectrum before pushing the PR?